web/src/main/java/com/bazaarvoice/emodb/web/resources/blob/BlobStoreResource1.java

```diff
@@ -412,14 +409,13 @@ public SuccessResponse put(@PathParam("table") String table,
 
         // The "ttl" query param can be specified to delete the blob automatically after a period of time
         Duration ttl = (ttlParam != null) ? ttlParam.get() : null;
-
-        // Perform the put
-        _blobStore.put(table, blobId, onceOnlySupplier(in), attributes, ttl);
-        
         if (null != ttl) {
-            _blobWritesWithTtl.mark();
+            throw new IllegalArgumentException(String.format("Ttl:{} is specified for blobId:{}"));
```

Comment:
Would a better error message be `Ttl:{} is no longer supported for blobId:{}` I'll leave this up to you, as I don't feel particularly strongly about it.